index.html

```diff
@@ -0,0 +1,208 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Lato:wght@300;400&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Gothic+A1&family=Inter&family=Lato:wght@300;400&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+rel="stylesheet">
+    
+    
+    <title>Electromagnetics Summit 2023</title>
+</head>
+<body>
+    
+
+    <nav class="desktop navbar navbar-dark bg-dark" >
+        <div class="container text-white">
+            <div class="navbar-brand"></div>
+            <ul class="d-flex " style="margin:0">
+                
+                <li class="nav-item">
+                  <a class="nav-link active text-white" aria-current="page" href="#"><img src="./images/facebook.png"> </a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-white" href="#"><img src="./images/twitter.png"></a>
+                </li>
+                <li class="nav-item ">
+                  <a class="nav-link text-white" href="#">English</a>
+                </li>
+                <li class="nav-item ">
+                    <a class="nav-link text-white" href="#">My page</a>
+                </li>
+
+                <li class="nav-item ">
+                    <a class="nav-link text-white" href="#">Log out</a>
+                </li>
+
+            </ul>
+        </div>
+    </nav>
+
+    <nav class="desktop navbar navbar-dark bg-white">
+        <div class="container text-white d-flex justify-content-between">
+            <a href="./index.html" class="navbar-brand"><img src="./images/my_logo.png"></a>
+            <a href="./about.html" class="navbar-brand" id="about" style="color: red" >About</a> 
+            <a href="#" class="navbar-brand" id="about" style="color: black" >Program</a> 
+            <a href="#" class="navbar-brand" id="about" style="color: black" >Join</a> 
+            <a href="#" class="navbar-brand" id="about" style="color: black" >Sponsor</a> 
+            <a href="#" class="navbar-brand" id="about" style="color: black" >News</a>
+            <a href="#" class="navbar-brand" id="about" style="color: red; border: solid 3px red; "  >CC Campaign</a>
+            
+        </div>
+    </nav>
+
+    <nav class="mobile navbar navbar-dark bg-white">
+        <div class="container text-white d-flex justify-content-between">
+            <a href="#" class="navbar-brand" id="hambuger"><img src="./images/mobile_menu.png"></a> 
+            <a href="./index.html" class="navbar-brand"><img src="./images/my_logo.png"></a>
+        </div>
+    </nav>
+
+    <div class="wrapper-all">
+        <div class="background-0"> 
+            <div class="wrapper-1">            
+
+                <div class="salute">"Hello! Sharing world"</div>
```

Comment:
- Kindly use semantic tags and ignore `div` as possible because the `div` is only usable for unknown tags 👍👌
- I think it is better to use `main` the separate each part with `section` to make your core readable 👍